js/client-teambuilder.js

```diff
@@ -796,6 +796,13 @@
 			this.exportMode = true;
 			this.update();
 		},
+		pokepasteExport: function () {
+			console.log("HI");
+			document.getElementById("pasteData").value = BattleLog.escapeHTML(Storage.exportTeam(this.curSetList));
+			document.getElementById("pasteTitle").value = BattleLog.escapeHTML(this.curTeam.name);
+			document.getElementById("pasteAuthor").value = BattleLog.escapeHTML(app.user.get('name'));
```

Comment:
You don't need the `escapeHTML` because you're setting it directly rather than through HTML.

Reply:
```suggestion
			document.getElementById("pasteData").value =Storage.exportTeam(this.curSetList);
			document.getElementById("pasteTitle").value =this.curTeam.name;
			document.getElementById("pasteAuthor").value =app.user.get('name');
```